homeassistant/components/alexa/handlers.py

```diff
@@ -234,6 +237,8 @@ async def async_api_turn_off(
         service = climate.SERVICE_TURN_OFF
     elif domain == fan.DOMAIN:
         service = fan.SERVICE_TURN_OFF
+    elif domain == remote.DOMAIN:
+        service = remote.SERVICE_TURN_OFF
```

Comment:
You need some tests for the handler code as well

Reply:
I have just added that.